contracts/RewardManager.sol

```diff
@@ -205,6 +207,34 @@ contract RewardManager is Ownable, Versionable, Safe {
     emit RewardSafeTransferred(msg.sender, oldOwner, newOwner);
   }
 
+  function withdrawFromRewardSafe(
+    address token,
+    uint256 value,
+    uint256 safeTxGas,
+    uint256 baseGas,
+    uint256 gasPrice,
+    address gasToken,
+    bytes calldata signature
+  ) external {
+    address rewardSafeOwner = getRewardSafeOwner(msg.sender);
+    bytes memory conSignature = contractSignature(msg.sender, rewardSafeOwner);
+    signatures[keccak256(conSignature)] = true;
+
+    execTransaction(
```

Comment:
@alex-cardstack but the safe will not trigger this right now if im not mistake because there is no signature allowing thru to the token contract.

Reply:
@tintinthong I am having problems getting this part to work - can you explain what you mean "there is no signature allowing thru to the token contract"?

Is that no the `signature` param to this function?

Reply:
Sure. All safe execTransactions will go to the `isValidSignature` callback; this is the case for the execTransaction within this contract. I will comment at the code

Reply:
OK nice, I will try that, thank you

Reply:
Thanks, that was the push in the right direction I needed. I have a further comment on my implementation there